measures/SimulationOutputReport/tests/simulation_output_report_test.rb

```diff
@@ -20,6 +21,7 @@ def test_SFD_1story_FB_UA_GRG_MSHP_FuelTanklessWH
                          "Door Area (ft^2)"=>30,
                          "Duct Surface Area (ft^2)"=>1440,
                          "Size, Heating System (kBtu/h)"=>60, # hp, not backup
+                         "Size, Heating Supplemental System (kBtu/h)"=>200, # backup
```

Comment:
This isn't correct. The OSW has `"supplemental_capacity" : "100.0"`.

Reply:
I was summing the living and basement.